manifests/t/TranslucentTB/TranslucentTB/2024.1/TranslucentTB.TranslucentTB.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TranslucentTB.TranslucentTB
+PackageVersion: "2024.1"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: TranslucentTB.exe
+  PortableCommandAlias: translucenttb
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```
Sorry, I forgot to hit `Start a review`.